src/bin/stratis_dumpmetadata.rs

```diff
@@ -48,14 +67,19 @@ fn run(devpath: &str) -> Result<(), String> {
 }
 
 fn main() {
-    let args: Vec<String> = env::args().collect();
-    if args.len() < 2 {
-        eprintln!("Usage: stratis_dumpmetadata <device>");
-        process::exit(2);
-    }
-    let devpath = &args[1];
+    let matches = App::new("stratis-dumpmetadata")
+        .arg(
+            Arg::with_name("dev")
+                .required(true)
+                .help("Print metadata of given device"),
+        )
+        .arg(Arg::with_name("print_bytes").help("Print byte buffer of device"))
```

Comment:
See the example here: https://github.com/stratis-storage/stratisd/blob/develop-2.3.0/src/bin/stratis-utils.rs#L152 for a way to handle print_bytes so that it is the familiar option, "--print_bytes".

Reply:
It is necessary to call the ```long()``` function in this case.

Sample help text for stratis-predict-usage:

```
$ ~/junk/usr/bin/stratis-predict-usage --help
stratis-predict-usage 

USAGE:
    stratis-predict-usage [FLAGS] [blockdevs]...

FLAGS:
        --encrypted    Whether the pool will be encrypted
    -h, --help         Prints help information
    -V, --version      Prints version information

ARGS:
    <blockdevs>...    Devices to include in the pool
```

you'll want to aim for something similar wrt. ```--print-bytes``` option.